quipucords/api/source/tests_source.py

```diff
@@ -279,6 +325,14 @@ def test_create_vc_with_hosts(self):
              'hosts': ['1.2.3.4', '1.2.3.5'],
              'credentials': [self.vc_cred_for_upload]})
 
+    def test_create_sat_with_hosts(self):
+        """A satellite source cannot have a host."""
```

Comment:
Is this supposed to read ` """A satellite source must not have multiple hosts."""` ?
Looks like same doc string is in above vcenter test